core/flyout_horizontal.ts

```diff
@@ -382,22 +382,10 @@ export class HorizontalFlyout extends Flyout {
         }
       }
 
-      if (
```

Comment:
It makes way more sense for this logic to be in the metrics manager, so I like that change. But I'm wondering if it has the possibility to break people who depend on the current values the metrics manager returns / will they have to update their math to account or not account for the flyout now? or was the translate added by the flyout already accounted for in the metrics?

Reply:
The translation gets overwritten the next time the metrics are applied to the workspace. So we were usually overwriting this update anyway.

We definitely /could/ break people that are depending on the current metrics values. But it's unlikely to do that because we're essentially adding new functionality. We have two new possible states which are categories + always open flyout and no categories + hiding flyout, and we're properly reporting those metrics.

Reply:
We should definitely manually test whether this breaks the continuous flyout though. I created a testing task in the GH project and assigned it to Rachel since she's doing release stuffs.